### DISCUSSION TITLE
Github Education Pack is requiring the wrong email from my university

### DISCUSSION THREAD

**Discussion by gabriel-accetta** [community/community#51338] [GitHub Education]
I study Information Systems at UniLaSalle-RJ and I've been trying to be approved for the GitHub Student Developer Pack since last year, on the first attempt I sent a photo of my college card, as it didn't have any kind of date I understood why the request it was rejected. On the second attempt I sent a printscreen of the student portal translated to English with a date and it was rejected again. On the third, and so far last attempt, I sent a photo of the most recent bank billet sent by the institution, with all the information required to prove my academic status, and it was rejected again, with the following return message:

> Your application cannot be reviewed until you fix the following:
> - Your image contains keywords indicating that it may be an acceptance letter. Only proof of current academic enrollment is acceptable.
> - You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

The billet contains all the information that this message says it doesn't have, and it's not possible that I am not near any campus of the institution since I'm inside the campus at the moment.

I found out that my school-issued email ( gabriel.accetta@soulasalle.com.br ) is being associated only with the High school section of my instituition: `Instituto Abel (La Salle Abel - Colégio em Niterói)`, but I am studying at the University section: `La Salle University Rio de Janeiro (Universidad La Salle) (UniLasalle)`

When I try to request it with my primary email, since it doesnt automatically choose the school, it returns me the following message:
![image.png](https://github.zendesk.com/attachments/token/wWYvUC3GsDngibs5QOO45y6l0/?name=image.png)

The problem is that the @lasalle.org.br and @unilasalle.edu.br are not the emails given to the students at UniLasalle-RJ, the email given to us is @soulasalle.com.br.

I know the problem is pretty confusing but this GitHub Student benefits would be really important to me, so I am doing everything I can to be accepted. Thanks in advance.

**Comment by samus-aran** [comment]
Hi @gabscrobson 👋 

Thank you for bringing this to our attention. We will pass this along to our internal team to investigate and updates will be posted on this thread 👍

**Reply by samus-aran** [reply]
Hi @gabscrobson 

Can you explain what happens if you selected your soulasalle.com.br address and _La Salle University Rio de Janeiro (Universidad La Salle) (UniLasalle)_. Confirmed this domain is associated with this school so you should be able to proceed. You may need to logged out and log in again. 👍

**Reply by gabriel-accetta** [reply]
Previously, it wouldn't let me choose _La Salle University Rio de Janeiro (Universidad La Salle) (UniLasalle)_ when the soulasalle.com.br address was chosen, since it was automatically selecting _Instituto Abel (La Salle Abel - Colégio em Niterói)_ for me. I just tried it again and this time it selects _La Salle University Rio de Janeiro_, as it should be. Now I'm going to send the request again and see what happens.

**Reply by gabriel-accetta** [reply]
Just sended it and I already got approved, thank you 👍 <3

**Reply by samus-aran** [reply]
That's awesome @gabscrobson  ! 🎉 Thanks so much for coming back and sharing this update. This will help other community members seeking similar answers. 

Start enjoying GitHub Global Campus 🌟:

- Connect with a local Campus Expert. For more information on campus experts, see "[About Campus Experts](https://education.github.com/students/experts)."
- Explore and claim offers for free industry tools from the [Student Developer Pack](https://education.github.com/pack).
- See upcoming in-person and virtual events for students, curated by GitHub Education and student leaders.
- View assignments from [GitHub Classroom](https://classroom.github.com/) with upcoming due dates.
- Stay in the know with [Campus TV](https://www.twitch.tv/githubeducation) episodes. Campus TV is created by GitHub and student community leaders and can be watched live or on demand.
- Discover and contribute to student-created repositories from GitHub Community Exchange. For more information, see "[About GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange)."

It's great to have you in the GitHub Education community and we hope to see you around 👋
